modules/tinymce/src/core/main/ts/api/EditorManager.ts

```diff
@@ -360,7 +360,7 @@ const EditorManager: EditorManager = {
       if (Env.browser.isIE() || Env.browser.isEdge()) {
         ErrorReporter.initError(
           'TinyMCE does not support the browser you are using. For a list of supported' +
-          ' browsers please see: https://www.tinymce.com/docs/get-started/system-requirements/'
+          ' browsers please see: https://www.tiny.cloud/docs/tinymce/6/support.html#_supported_web_browsers'
```

Comment:
Another ID that should be fixed (i.e. not using an underscore prefixed ID)

- [ ] Fix this here and in the docs to use a fixed ID